plugin.cmake

```diff
@@ -113,7 +113,12 @@ add_histomicstk_python_test(blob_detection_filters
 
 add_histomicstk_python_test(cli_common
   # Work around CMake bug when using the same image multiple times
-  # EXTERNAL_DATA "plugins/HistomicsTK/Easy1.png"
+  EXTERNAL_DATA
+  # "plugins/HistomicsTK/Easy1.png"
```

Comment:
The `# Work around CMake bug` comment ought to be right before the commented-out item